src/Utils.php

```diff
@@ -588,15 +597,167 @@ public static function generate_shipping_options_html( $backend_data ) {
+				'parcel_non_eu'         => esc_html__( 'Parcels non-EU Insured Plus', 'postnl-for-woocommerce' ),
+			);
+		}elseif($country == 'eu'){
+			return array(
+				'standard_shipment'     => esc_html__( 'Standard shipment', 'postnl-for-woocommerce' ),
+				'id_check'              => esc_html__( 'ID Check', 'postnl-for-woocommerce' ),
+				'return_no_answer'      => esc_html__( 'Return if no answer', 'postnl-for-woocommerce' ),
+				'signature_on_delivery' => esc_html__( 'Signature on Delivery', 'postnl-for-woocommerce' ),
+				'only_home_address'     => esc_html__( 'Only Home Address', 'postnl-for-woocommerce' ),
+				'letterbox'             => esc_html__( 'Letterbox', 'postnl-for-woocommerce' ),
+				'packets'               => esc_html__( 'Packet Untracked', 'postnl-for-woocommerce' ),
+				'standard_belgium'      => esc_html__( 'Standard Shipment Belgium', 'postnl-for-woocommerce' ),
+				'mailboxpacket'         => esc_html__( 'Boxable Packet Untracked', 'postnl-for-woocommerce' ),
+				'eu_parcel'             => esc_html__( 'Parcels EU Track & Trace Insured', 'postnl-for-woocommerce' ),
+				'insured_shipping'      => esc_html__( 'Insured Shipping', 'postnl-for-woocommerce' ),
+				'insured_plus'          => esc_html__( 'Insured Plus', 'postnl-for-woocommerce' ),
+			);
+		}else{ 
+			return array(
+				'standard_shipment'     => esc_html__( 'Standard shipment', 'postnl-for-woocommerce' ),
+				'id_check'              => esc_html__( 'ID Check', 'postnl-for-woocommerce' ),
+				'return_no_answer'      => esc_html__( 'Return if no answer', 'postnl-for-woocommerce' ),
+				'signature_on_delivery' => esc_html__( 'Signature on Delivery', 'postnl-for-woocommerce' ),
+				'only_home_address'     => esc_html__( 'Only Home Address', 'postnl-for-woocommerce' ),
+				'letterbox'             => esc_html__( 'Letterbox', 'postnl-for-woocommerce' ),
+				'packets'               => esc_html__( 'Packet Untracked', 'postnl-for-woocommerce' ),
+				'standard_belgium'      => esc_html__( 'Standard Shipment Belgium', 'postnl-for-woocommerce' ),
+				'mailboxpacket'         => esc_html__( 'Boxable Packet Untracked', 'postnl-for-woocommerce' ),
+				'parcel_non_eu'     	=> esc_html__( 'Parcels Non-EU Track & Trace', 'postnl-for-woocommerce' ),
+				'insured_shipping'      => esc_html__( 'Insured Shipping', 'postnl-for-woocommerce' ),
+				'insured_plus'          => esc_html__( 'Insured Plus', 'postnl-for-woocommerce' ),
+				'eu_parcel'             => esc_html__( 'Parcels EU Track & Trace', 'postnl-for-woocommerce' ),
+			);
+		}
+	}
+
+	/**
+	 * Check if current order/cart is eligible for automatically use letterbox.
+	 *
+	 * @param \WC_Order|\WC_Cart|int $order \WC_order, \WC_Cart or Order ID.
+	 *
+	 * @return boolean
+	 */
+	public static function is_eligible_auto_letterbox( $order ) {
+
+		// Check order
+		if ( is_int( $order ) ) {
+			$order = wc_get_order( $order );
+		}
+		if(wc_get_base_location()['country'] == 'BE'){
+			return false;
+		}
```

Comment:
```suggestion
		if ( 'BE' !== wc_get_base_location()['country'] ) {
			return false;
		}
```
[When doing logical comparisons involving variables, always put the variable on the right side and put constants, literals, or function calls on the left side](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#yoda-conditions)